data/analytics/events.yml

```diff
@@ -784,12 +813,18 @@
         value: event_data
       - name: event_data
         value:
+        - name: event_name
+          value: select_content
         - name: action
           value: opened/closed
-        - name: event_name
         - name: index
-        - name: index_total
+          value:
+          - name: index_section
+            value: The section that this step is in. The section in this case refers to the step number.
```

Comment:
This is a good addition. However I think it should go into the data entry for the `index_section`. Unless it's changed in the move to dev docs, if no value is supplied in an event for an attribute, it'll pull through the value from the attribute data. That means for something generic like this we could have this description in one place and it'll propagate to all events that include it.

Reply:
(same for `index_section_count`)

Reply:
When I remove the value, instead it pulls in the `type`. So it just becomes 
```yml
index_section: number
index_section_count: number
```

Is this OK?

Reply:
I've added some extra context to `index_section` and `index_section_count` in `attributes.yml` but if you think it isn't needed I can remove it :+1:

Reply:
Ah yes, starting to remember how this works! Yes I think it's fine that it just says `number` and that users can then click on the attribute to find out more information about what that number represents. Also good other additions 👍